memcached-image/openSUSE_Tumbleweed.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<param>
+  <container_base_image>obsrepositories:/opensuse/tumbleweed</container_base_image>
+  <container_name>opensuse/memcached</container_name>
+  <container_tag>latest</container_tag>
```

Comment:
should this be latest ?

Reply:
Important question for me.

Reply:
I think for Tumbleweed, yes. But I haven't thought about it in detail. What are the possible problems you see with latest?

Reply:
We need to publish images with a specific tag (and not always "latest"), else k8s won't have a way to deal with rollbacks and many other specific behaviour triggering when "latest" is used.

Reply:
That's why we have `additionaltags="%LONG_VERSION%,%LONG_VERSION%-%RELEASE%"`

Reply:
ok good.